tools/c7n_azure/c7n_azure/resources/appserviceplan.py

```diff
@@ -118,6 +118,7 @@ class ResizePlan(AzureBaseAction):
 
     schema = utils.type_schema(
         'resize-plan',
+        required=['size'],
```

Comment:
Did you open a bug for Rest API swagger? 

Since this is a PATCH they should change the behavior of their backend or update swagger to reflect that SKU is required.

Reply:
Maybe their PATCH does work - we used to be passing the wrong fields apparently...

Reply:
PATCH does work, but the model from the API spec appears to be wrong. I opened a bug [here](https://github.com/Azure/azure-rest-api-specs/issues/7139)